tests/integration/run.py

```diff
@@ -87,7 +87,7 @@ def configure_logging(verbose: bool) -> None:
 
 DEFAULT_SCHEMAS: dict[str, Sequence[str]] = {
     "output_json_report": ["tests", "schema_validation", "report_schema.json"],
-    "find_source_json_report": ["tests", "repo_finder", "resources", "find_source_report_schema.json"],
+    "find_source_json_report": ["src", "macaron", "resources", "schemastore", "find_source_report_schema.json"],
```

Comment:
Do we need to remove the old file at `["tests", "repo_finder", "resources", "find_source_report_schema.json"]` ?

Reply:
I have already removed this file.